adagrams/game.py

```diff
@@ -1,13 +1,107 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    pile = []
+    hand = []
+    # created a list of letters
+    for k, v in LETTER_POOL.items():  
```

Comment:
Prefer to use more descriptive variables because 'k' and 'v' are too generic. I can guess that `k` means key and `v` means value, but what do these variables reference?

The key in this dictionary references a letter and the value is the frequency of the letter.

```suggestion
    for letter, frequency in LETTER_POOL.items():  
```